.tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml

```diff
@@ -36,4 +36,30 @@ spec:
         - name: ownerName
           value: $(context.pipelineRun.name)
         - name: ownerUid
-          value: $(context.pipelineRun.uid)
\ No newline at end of file
+          value: $(context.pipelineRun.uid)
+    - name: ols-install
+      description: Task to install bundle onto ephemeral namespace
+      runAfter:
+        - eaas-provision-space
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: SecretRef
+          value: $(task.eaas-provision-space.results.SecretRef)
+        - name: revision
+          value: "{{revision}}"
+      taskSpec:
+        params:
+          - name: SNAPSHOT
+          - name: SecretRef
+        steps:
+          - name: install-operator
+            env:
+              - name: SNAPSHOT
+                value: $(params.SNAPSHOT)
+            image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/bundle:on-pr-5690274f2c4ea8358eb73b942074c1dff684b723
```

Comment:
will it remain the same?

Reply:
no, for sure no. That's why I started the thread to see if we can trigger the bundle building in every PR
Then I could just have quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/bundle:on-pr-{{revision}}